src/fsharp/fsc.fs

```diff
@@ -699,17 +718,16 @@ module XmlDocWriter =
 // cmd line - option state
 //----------------------------------------------------------------------------
 
-let getModuleFileName() = 
-    Path.Combine(System.AppDomain.CurrentDomain.BaseDirectory,
-                           System.AppDomain.CurrentDomain.FriendlyName)  
-
-let defaultFSharpBinariesDir = Filename.directoryName (getModuleFileName())
 
+let defaultFSharpBinariesDir = 
+#if FX_RESHAPED_APPCONTEXT
```

Comment:
AppContext has not been reshaped AFAIK, even on desktop it has BaseDirectory.  Should this just be `FX_NO_APPDOMAINS` ?